src/kernel/src/obj/pages.rs

```diff
@@ -5,22 +5,21 @@ use twizzler_abi::device::{CacheType, MMIO_OFFSET};
 use x86_64::{PhysAddr, VirtAddr};
 
 use crate::{
-    arch::memory::phys_to_virt,
-    memory::frame::{self, Frame, PhysicalFrameFlags},
+    arch::memory::{frame::FRAME_SIZE, phys_to_virt},
+    memory::frame::{self, FrameRef, PhysicalFrameFlags},
 };
 
 use super::{Object, PageNumber};
 
-bitflags::bitflags! {
-    pub struct PageFlags:u32 {
-        const WIRED = 1;
-    }
+#[derive(Debug)]
+enum FrameOrWired {
+    Frame(FrameRef),
+    Wired(PhysAddr),
```

Comment:
Can you document what `Wired` is supposed to mean?

Reply:
Yup!